Source-Code/Drone2/Drone2.ccl

```diff
@@ -0,0 +1 @@
+VERSION=20110520

```

Comment:
Do we need to upload this file?  If not we can just have them on the .gitignore (if they aren't already) so we don't have extra unnecessary files in the repo.